DataFormats/MuonReco/src/classes_def.xml

```diff
@@ -177,5 +177,25 @@
   <class name="edm::PtrVector<reco::Muon>" />
   <class name="edm::Wrapper<edm::PtrVector<reco::Muon> >" />
 
+  <selection>
+    <class name="std::vector<EmulatedME0Segment>"/>
+    <class name="EmulatedME0Segment" ClassVersion="12">
+      <version ClassVersion="12" checksum="1065224607"/>
```

Comment:
these are new classes without past history of being stored in public areas: drop all "past" versions
and start with version 2 instead
```diff
 +    <class name="EmulatedME0Segment" ClassVersion="2"> 
 +      <version ClassVersion="2" checksum="1065224607"/> 
 +    </class> 
```

same for reco::ME0Muon